Quiz Master/src/screens/options/ScreenTheFirst.tsx

```diff
@@ -0,0 +1,121 @@
+import "./ScreenTheFirst.css";
+import ChangePageButton from "../../components/ChangePageButton";
+import { ChangeEvent, useState } from "react";
+import { DescriptiveSelectList } from "../../components/DescriptiveSelectList";
+import { difficulties, types, times } from "../../constants/ValueOptions";
+import { Difficulty, QuestionCounts, QuestionType } from "../../types/Question";
+import { useDispatch, useSelector } from "react-redux";
+import { playAudio, Sounds } from "../../store/slices/audioPlayerSlice";
+import { Routes } from "../../constants/routes";
+import { setGameSettings } from "../../store/slices/settingsSlice";
+import { AppDispatch, RootState } from "../../store";
+import { startGameThunk } from "../../store/slices/gameSlice";
+
+const ScreenTheFirst = () => {
+  const dispatch: AppDispatch = useDispatch();
+  const settings = useSelector((state: RootState) => state.settings);
```

Comment:
Consider extracting all your selectors in your redux part of the app, where you would have something like the following:
const settingsSelector = (state: RootState) => state.settings;

And then you can reuse that across your project